elements/pl-rich-text-editor/pl-rich-text-editor.js

```diff
@@ -34,3 +34,45 @@ window.PLRTE = function (uuid, options) {
     );
   });
 };
+
+// Override default implementation of 'formula'
+
+var Embed = Quill.imports.parchment.Embed;
+
+class MathFormula extends Embed {
+  static create(value) {
+    const node = super.create(value);
+    if (typeof value === 'string') {
+      this.waitUntilLoaded(node, value);
+    }
+    return node;
+  }
+
+  static waitUntilLoaded(node, value) {
```

Comment:
I'd recommend naming this function something other than `waitUntilLoaded` - maybe `updateNode`? That's the main this function does, the waiting is just am implementation detail IMO.

Related: please add an inline comment explaining _why_ waiting is necessary. I saw that in the PR description, but we really want that to be located in the source code so it's easy for future maintainers to discover.